src/Core/Geographic/Crs.ts

```diff
@@ -0,0 +1,185 @@
+import proj4 from 'proj4';
+
+import type { ProjectionDefinition } from 'proj4';
+
+proj4.defs('EPSG:4978', '+proj=geocent +datum=WGS84 +units=m +no_defs');
+
+/**
+ * A projection as a CRS identifier string.
+ */
+export type ProjectionLike = string;
```

Comment:
Yeah, we should export it in the documentation since this is the type of the argument of some functions of the `CRS` module.

I think the documentation is complete, this is an identifier referencing an already defined projection (I should maybe add this to the documentation). In `isGeocentric`, the call to `proj4.defs` is only for retrieving an already defined projection as an object of type `proj4.ProjectionDefinition`.

Reply:
Moreover, I added such type alias since we'll surely want in a near future to have a `ProjectionLike` either as an identifier or an `proj4.ProjectionDefinition`.

Reply:
Oh ok thanks, `proj4.defs` is quite misleading :sweat_smile: 

I just checked and it is not exported in the doc though, you may need to add a @typedef tag like for CameraTransformOptions? : `@typedef {Object} CameraTransformOptions`

Also, I'm wandering if it should appear in the Crs doc page or have its own page or be in a `Types` page, WDYT?

Reply:
The equivalent of typedef in typescript or either an `interface` or a `type`. Instead of being in the documentation, we can write the types directly.

Reply:
Yes but except if I'm missing something it is still not exported in the doc (see http://localhost:8080/docs/out/#api/Geographic/CRS after running `npm run doc`).